### ISSUE TITLE
模型在多块图像拼接时存在边界伪影问题

### ISSUE THREAD

**Issue by flynn17625** [ICSResearch/UFC-Net#16]
1.问题描述
在使用 UFC-Net 对大于block_size（默认 32x32）的图像进行压缩感知重建时，图像分块拼接区域出现明显的边界伪影。具体表现为：
测试 Set11 数据集中的barbara.png（512x512）时，分块拼接处存在明暗交替的条纹
边界伪影在低压缩率（如 cs_ratio=0.04）时尤为明显
单块 32x32 图像重建无此问题，仅在多块拼接时出现
2.复现步骤
配置环境：Python 3.11.5 + PyTorch 1.12.0，使用默认参数
运行评估脚本：python eval.py --rate 0.04 --device 0
观察./recon_img/Y/Set11_GREY/4/目录下的重建图像
重点查看 512x512 等大尺寸图像的分块拼接区域
3.问题定位
通过分析代码逻辑，推测问题根源在于：
eval.py中图像分块与拼接逻辑存在缺陷：
python
运行
# 分块处理
inputs = torch.cat(torch.split(inputs, split_size_or_sections=config.block_size, dim=3), dim=0)
inputs = torch.cat(torch.split(inputs, split_size_or_sections=config.block_size, dim=2), dim=0).to(config.device)

# 拼接处理
reconstruction = torch.cat(torch.split(reconstruction, split_size_or_sections=1 * idx, dim=0), dim=2)
reconstruction = torch.cat(torch.split(reconstruction, split_size_or_sections=1, dim=0), dim=3)
分块时未考虑边界信息交互，每个块独立重建导致边缘特征不连续
缺少跨块的一致性约束机制
4/建议解决方案
实现重叠分块重建策略，参考代码：
python
运行
# 改为重叠分块（示例）
overlap = 4  # 重叠像素数
patches = []
for i in range(0, h, config.block_size - overlap):
    for j in range(0, w, config.block_size - overlap):
        patch = inputs[:, :, i:i+config.block_size, j:j+config.block_size]
        patches.append(patch)
# 重建后使用加权平均融合重叠区域
在FPC模型中添加跨块注意力机制，增强全局一致性
调整utils/config.py中的block_size参数为可配置项，方便测试不同块大小的影响
附加信息
系统环境：Ubuntu 20.04 + CUDA 11.6
测试图像：Set11 中的barbara.png和boat.png
